src/lang-promql/client/prometheus.ts

```diff
@@ -68,6 +68,7 @@ export interface PrometheusConfig {
   fetchFn?: FetchFn;
   // cache will allow user to change the configuration of the cached Prometheus client (which is used by default)
   cache?: CacheConfig;
+  httpMethodUsed?: 'post' | 'get';
```

Comment:
Would it be better to standardize on the upper-cased `POST` and `GET`, as the official examples in https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch and the fetch spec (https://fetch.spec.whatwg.org/#methods) use?

Reply:
yeah totally make sense. I don't even know why I didn't put it already in uppercase. It's a bit a common definition across the different language.